chia/server/rate_limits.py

```diff
@@ -92,6 +92,9 @@ class RLSettings:
     ProtocolMessageTypes.request_header_blocks: RLSettings(500, 100),
     ProtocolMessageTypes.reject_header_blocks: RLSettings(100, 100),
     ProtocolMessageTypes.respond_header_blocks: RLSettings(500, 2 * 1024 * 1024, 100 * 1024 * 1024),
+    ProtocolMessageTypes.request_header_block_blobs: RLSettings(500, 100),
+    ProtocolMessageTypes.reject_header_block_blobs: RLSettings(100, 100),
+    ProtocolMessageTypes.respond_header_block_blobs: RLSettings(500, 2 * 1024 * 1024, 100 * 1024 * 1024),
```

Comment:
I don't think we need new APIs for this. The existing APIs always serialize everything already. The only difference you make here is that you serialize the block as a blob of bytes (which in-turn is a serialized FullBlock). It would seem reasonable to just include the serialized blob verbatim into the existing response instead.

Reply:
You do need a new request because the current one expects a filter in the response (for old protocol)

Reply:
an alternative would be to accept an optional flag to indicate that you don't need the filter in the response. But maybe that's more complicated (although, it would make it backwards compatible)

Reply:
Not sure how to add an optional argument, this is not really possible with streamable protocol right now.